docs/user-guide/configuration/annotations.md

```diff
@@ -37,6 +37,7 @@ Some Annotations are used to modify the properties of the build environment. The
 | screwdriver.cd/executor | Ask your cluster admin | This will determine what compute system is used to run the build. For example, set the build to run in a VM, a kubernetes pod, a docker container, or a Jenkins agent. |
 | screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` / `TURBO` | When using a `k8s` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`), 6 (`HIGH`) and 12 (`TURBO`) CPU resources. Default is `LOW`.<br>   When using a `k8s-vm` executor, this will allow the user to choose between 1 (`MICRO`), 2 (`LOW`), 6 (`HIGH`), and 12 (`TURBO`) CPU resources. Default is `LOW`. |
 | screwdriver.cd/disk | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 20 GB (`LOW`) and 50 GB (`HIGH`) disk space. Default is `LOW`. |
+| screwdriver.cd/diskSpeed | `LOW` / `HIGH` / `TURBO` | When using a `k8s-vm` executor, this will allow the user to choose between machines with different disk speed. Default is `LOW`. |
```

Comment:
is there any hard numbers to match the different settings?

Reply:
there won't be hard number, it's more like the type of the disks, e.g. ssd or SATA etc. And different cluster may have different standard, that depends on how they label their machines